internal/server/rest.go

```diff
@@ -184,6 +184,8 @@ func createRestServer(ip string, port int, needToken bool) *http.Server {
 	r.HandleFunc("/data/import", configurationImportHandler).Methods(http.MethodPost)
 	r.HandleFunc("/data/import/status", configurationStatusHandler).Methods(http.MethodGet)
 	r.HandleFunc("/connection/websocket", connectionHandler).Methods(http.MethodGet, http.MethodPost, http.MethodDelete)
+	r.HandleFunc("/async/task", asyncTaskHandler).Methods(http.MethodGet, http.MethodPost)
```

Comment:
How about data/import/async? Or async/data/import? By this way, each task can have static body and easy to understand.

Reply:
updated.